### ISSUE TITLE
Make “case origin” modifiable and integrate it into the case directory + in the export

### ISSUE THREAD

**Issue by carolinverset** [hzi-braunschweig/SORMAS-Project#4499]
### Situation Description

To date, the variable "case origin" can only be modified when the case is created, whether by import or by entering within the application. Then, this field is no longer editable by the user.
 
![image](https://user-images.githubusercontent.com/65449450/108714446-d5691d00-7519-11eb-88c4-c680e2a6e7d5.png)

### Feature Description
- Make this variable modifiable within the application 
- Include it in the case directory (new additional column + filter)
- Integrate it into the various exports (basic and detailed)

### Additional Information
@ftavin @bernardsilenou @emond

**Comment by MateStrysewske** [comment]
@carolinverset What's the reason for making this field configurable?

**Comment by bernardsilenou** [comment]
@MateStrysewske I think the main change here is to make case origin editable after case creation.
Similar need request was raised here: #2897

**Comment by ftavin** [comment]
@MateStrysewske What's the reason for making this field not-editable in the first place ? is there technical reasons ?

**Comment by bernardsilenou** [comment]
@ftavin The reason I can think of was that in most African countries, point of entry (POE boarder to the country, region, etc) do monitor cases at boarders (airport, sea port, land crossing, etc)  but they are not a surveillance unit to manage the cases. Their main goal is to identify a sick person, create a case and refer the case to a district where the case can be counted by assigning EPID to them.
After the case is refereed to a hospital, then we wanted to be able to track this process and prevent users who then continue managing the case from mistakenly editing the data.
To me, this management and referral process of POE cases was the main reason, not a technical. @MateStrysewske  can validate or add more points.

**Comment by MateStrysewske** [comment]
@ftavin Partly, I think it's not really trivial because point of entry cases work slightly different than usual cases.

@bernardsilenou Let me rephrase my question - why is it necessary? I understood what the purpose of this issue is, but I don't really see a reason for it. The only valid reason that would come to my mind would be accidentally entered data, but in that case the case could simply be deleted and re-created as an in-country case (or the other way around).

In any case, I'm sure it's perfectly doable to make this field editable, but it might, as written above, not be trivial. So at least I'd suggest to assign this issue a very low priority, unless there's a different purpose than the one I just outlined of course.

**Comment by bernardsilenou** [comment]
@MateStrysewske 
- I can not tell if this is an urgent need we need to worry about,  @carolinverset can tell us more.
- The only situation that i think such a need can arise is if a user mistakenly create a case as POE case, contacts and 
followup is documented and a supervisor latter realized it was an error at data creation. At this point users may not want to delete the case and update all the data. This is a rare situation, so I can not tell if it is worth the development effort.

**Comment by carolinverset** [comment]
@MateStrysewske @bernardsilenou @ftavin 
Sorry for this late response. I approached ARS Martinique which is at the origin of this request for further information. The difficulty comes from the French mode of organization which uses this field by import. The use of this field is being diverted to allow users to know if a case has traveled in the past two weeks. We are continuing our reflection with ARS Martinique and then coming back to you.

**Comment by carolinverset** [comment]
After discussion with the user, we found another solution to meet this need.
